include/seqan3/range/views/to.hpp

```diff
@@ -0,0 +1,26 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \brief Provides seqan3::views::to.
+ */
+
+#pragma once
+
+#include <range/v3/view/zip.hpp>
```

Comment:
should this not be `#include <range/v3/range/conversion.hpp>`

Reply:
not resolved?

Reply:
Oh no, then more got lost in the rebases..